tasks/task6/schedule/schedule.py

```diff
@@ -0,0 +1,67 @@
+import time
+from sender import Sender
+from plataform import MessagePlataform
+
+class Schedule(Sender):
```

Comment:
Another pattern that can go along with your current design would be the chain of responsability pattern, as we have a chain of stages, and if the message has a response in 10 min the next action doesn't need to be executed, and if not the action should pass to the other handler and so on.